index.js

```diff
@@ -96,6 +107,50 @@ function parse(str, options) {
  */
 
 function serialize(name, val, options) {
+  if( typeof name === 'object') {
+    var cookies = name;
+    var serializeOptions = val;
+
+    var cookieNames = Object.keys(cookies);
+    if(0 === cookieNames.length) {
+      return '';
+    } else if(cookieNames.length > 1) {
+      // If there are more than one cookies to serialize, only allow
+      // an encoding option to be set
+      var opt = serializeOptions || {};
+      serializeOptions = {
```

Comment:
I could be wrong about this, but my thinking was that if we are given several cookies to serialize, they want those cookies in the `Cookie` request header which, from what I read in [RFC6265 Section 5.4](http://tools.ietf.org/html/rfc6265#section-5.4) should be serialized as the cookie name followed by an '=' followed by the cookie value with all cookies separated by a semicolon. So if we want to serialize multiple cookies into a single string, we should not allow additional cookie parameters in the string (again, I could be wrong about this, please correct me if so).

One possible option is: given an object with multiple cookies and an options object with cookie parameters we can return an array of serialized cookie strings, one for each cookie, with all the parameters set as could be used in a `Set-Cookie` header. Another option is to let the caller indicate in the options object whether they want their cookies serialized for a `Set-Cookie` response header or a `Cookie` request header.